staff/pablo-rey/spotify-template/styles/components/_brand.sass

```diff
@@ -0,0 +1,26 @@
+%brand__name--base
```

Comment:
@paulusrex nice use of heritance and varibales sass files. Keep ti up!

Reply:
Thanks